genesis/process/shardGenesisBlockCreator.go

```diff
@@ -310,21 +311,15 @@ func createProcessorsForShardGenesisBlock(arg ArgsGenesisBlockCreator, enableEpo
 	}
 
 	argsNewVMFactory := shard.ArgVMContainerFactory{
-		Config:                         arg.VirtualMachineConfig,
-		BlockGasLimit:                  math.MaxUint64,
-		GasSchedule:                    arg.GasSchedule,
-		ArgBlockChainHook:              argsHook,
-		EpochNotifier:                  epochNotifier,
-		DeployEnableEpoch:              arg.EpochConfig.EnableEpochs.SCDeployEnableEpoch,
-		AheadOfTimeGasUsageEnableEpoch: arg.EpochConfig.EnableEpochs.AheadOfTimeGasUsageEnableEpoch,
-		ArwenV3EnableEpoch:             arg.EpochConfig.EnableEpochs.RepairCallbackEnableEpoch,
-		ArwenChangeLocker:              genesisArwenLocker,
-		ESDTTransferParser:             esdtTransferParser,
+		Config:             arg.VirtualMachineConfig,
+		BlockGasLimit:      math.MaxUint64,
+		GasSchedule:        arg.GasSchedule,
+		ArgBlockChainHook:  argsHook,
+		EpochNotifier:      epochNotifier,
+		EpochConfig:        arg.EpochConfig.EnableEpochs,
+		ArwenChangeLocker:  genesisArwenLocker,
+		ESDTTransferParser: esdtTransferParser,
 	}
-	log.Debug("shardGenesisCreator: enable epoch for sc deploy", "epoch", argsNewVMFactory.DeployEnableEpoch)
```

Comment:
you removed these epoch activation prints. Can we have them in the NewVMContainerFactory function?

Reply:
I removed from shardGenesisBlock creator - but we still have the one from nodeRunner and from node factory. Here, at genesis is not needed.

Reply:
and these prints are not more relevant - as they were used in arwen 1.2 and 1.3.